pkg/agents/team2/actions.go

```diff
@@ -0,0 +1,64 @@
+package team2
+
+import (
+	"math/rand"
+
+	"github.com/SOMAS2021/SOMAS2021/pkg/utils/globalTypes/food"
+)
+
+func InitActionSpace() actionSpace {
+	//TODO: actionID might be removed in further versions
+	//index : 0 => disregard food
+	//index : 1 => satisfice with food
+	//index : 2 => satisfy with food
+	initialActionSpace := actionSpace{}
+	initialActionSpace.actionId = make([]int, 3)
+	for i := 0; i < 3; i++ {
+		initialActionSpace.actionId[i] = i
+	}
+	m := map[int]func(hp int) food.FoodType{
+		//actions based on the current hp level
+		initialActionSpace.actionId[0]: DisFood,
+		initialActionSpace.actionId[1]: Satisfice,
+		initialActionSpace.actionId[2]: Satisfy,
+	}
+	initialActionSpace.actionSet = m
+	return initialActionSpace
+}
+
+//Need to change this func when adding new actions
+
+func DisFood(hp int) food.FoodType {
+	return 0
+}
```

Comment:
What does this function actually do?

Reply:
For our MVP, the logic of this function is not implemented yet. Now it always returns zero as the amount of food to be taken.

Reply:
I assumed so, would be good if you could document the intended behaviour for future reference or at least mark it as a todo